lib/fluent/plugin/in_http.rb

```diff
@@ -84,6 +84,8 @@ class HttpInput < Input
     config_param :dump_error_log, :bool, default: true
     desc 'Add QUERY_ prefix query params to record'
     config_param :add_query_params, :bool, default: false
+    desc 'Handle Authroization header authentication with specified secret'
+    config_param :authorization_token, :string, default: nil, secret: true
 
```

Comment:
`:authorization_token` parameter name may be confused because it is  actually pair of  `<type> <credential>`.
`:authorization_fields` ? or something maybe better.

Reply:
It seems okay as is.